### PR TITLE
Update pl_lolokoko.m3u

### PR DESCRIPTION
**Pull request by GuyInCanada**

### CODE REVIEW

File: streams/pl_lolokoko.m3u [hunk 1]
```diff
@@ -1,45 +1,176 @@
-#EXTM3U
-#EXTINF:-1 tvg-id="CBeebies.pl",CBeebies (Poland) (576p)
-https://cdn.lolokoko.tv/CBeebies.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ActiveFamily.pl",Active Family (1080p)
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:1]:
```suggestion
#EXTM3U
#EXTINF:-1 tvg-id="ActiveFamily.pl",Active Family (1080p)
```

File: streams/pl_lolokoko.m3u [hunk 2]
```diff
@@ -1,45 +1,176 @@
...
+https://cdn-nvm-london.lolokoko.tv/amc.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="AnimalPlanet.pl",Animal Planet (720p)
+https://cdn-nvm-london.lolokoko.tv/ANIMALPLANET.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="AXNBlack.pl",AXN Black (720p)
+https://cdn-nvm-london.lolokoko.tv/axnblack.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="AXNWhite.pl",AXN White (720p)
+https://cdn-nvm-london.lolokoko.tv/axnwhite.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="BBCBrit.pl",BBC Brit (720p)
+https://cdn-nvm-london.lolokoko.tv/BBCBRITPOLSKA.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="BBCCBeebies.pl",BBC CBeebies (576p)
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:13]:
```suggestion
#EXTINF:-1 tvg-id="CBeebies.pl",BBC CBeebies (576p)
```

File: streams/pl_lolokoko.m3u [hunk 3]
```diff
@@ -1,45 +1,176 @@
-#EXTM3U
-#EXTINF:-1 tvg-id="CBeebies.pl",CBeebies (Poland) (576p)
-https://cdn.lolokoko.tv/CBeebies.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ActiveFamily.pl",Active Family (1080p)
+https://cdn-nvm-london.lolokoko.tv/ActiveFamily.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="AMC.pl",AMC (720p)
+https://cdn-nvm-london.lolokoko.tv/amc.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="AnimalPlanet.pl",Animal Planet (720p)
+https://cdn-nvm-london.lolokoko.tv/ANIMALPLANET.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:6]:
```suggestion
```
DMCA.

File: streams/pl_lolokoko.m3u [hunk 4]
```diff
@@ -1,45 +1,176 @@
...
+#EXTINF:-1 tvg-id="FightBox.pl",Fightbox (720p)
+https://cdn-nvm-london.lolokoko.tv/fightbox.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="FilmBoxArthouse.pl",Filmbox Arthouse (576p)
+https://cdn-nvm-london.lolokoko.tv/FilmBoxArthouse.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="FilmBoxFamily.pl",Filmbox Family (576p)
+https://cdn-nvm-london.lolokoko.tv/filmboxfamily.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="FilmBoxPremium.pl",Filmbox Premium (576p)
+https://cdn-nvm-london.lolokoko.tv/FilmBoxPremium.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="FoodNetwork.pl",Food Network (720p)
+https://cdn-nvm-london.lolokoko.tv/FoodNetwork.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:52]:
```suggestion
```
DMCA.

File: streams/pl_lolokoko.m3u [hunk 5]
```diff
@@ -1,45 +1,176 @@
...
+#EXTINF:-1 tvg-id="FOX.pl",Fox (720p)
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:53]:
```suggestion
#EXTINF:-1 tvg-id="Fox.pl",Fox (720p)
```

File: streams/pl_lolokoko.m3u [hunk 6]
```diff
@@ -1,45 +1,176 @@
...
+https://cdn-nvm-london.lolokoko.tv/FoxComedy.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="HISTORY2.pl",H2 (720p)
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:55]:
```suggestion
#EXTINF:-1 tvg-id="History2.pl",H2 (720p)
```

File: streams/pl_lolokoko.m3u [hunk 7]
```diff
@@ -1,45 +1,176 @@
...
+https://cdn-nvm-london.lolokoko.tv/H2.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="HBO.pl",HBO (720p)
+https://cdn-nvm-london.lolokoko.tv/HBO.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="HBO2.pl",HBO 2 (720p)
+https://cdn-nvm-london.lolokoko.tv/HBO2.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="HBO3.pl",HBO 3 (720p)
+https://cdn-nvm-london.lolokoko.tv/HBO3.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="HGTV.pl",HGTV (720p)
+https://cdn-nvm-london.lolokoko.tv/hgtv.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:64]:
```suggestion
```
DMCA.

File: streams/pl_lolokoko.m3u [hunk 8]
```diff
@@ -1,45 +1,176 @@
...
+https://cdn-nvm-london.lolokoko.tv/HISTORY.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="KinoTV.pl",Kino TV (720p)
+https://cdn-nvm-london.lolokoko.tv/filmbox.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="KinoPolskaMuzyka.pl",Kinopolska (720p)
+https://cdn-nvm-london.lolokoko.tv/Kinopolska.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="MiniMini+.pl",MiniMini+ (720p)
+https://cdn-nvm-london.lolokoko.tv/MiniMini.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="MTVLive.pl",MTV Live
+https://cdn-nvm-london.lolokoko.tv/MTVLIVE.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="MTVPolska.pl",MTV Polska (720p)
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:75]:
```suggestion
#EXTINF:-1 tvg-id="MTV.pl",MTV Polska (720p)
```

File: streams/pl_lolokoko.m3u [hunk 9]
```diff
@@ -1,45 +1,176 @@
...
+https://cdn-nvm-london.lolokoko.tv/polsatsport.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PolsatViasatHistory.pl",Polsat Viasat History (720p)
+https://cdn-nvm-london.lolokoko.tv/VIASATHISTORY.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PolsatViasatNature.pl",Polsat Viasat Nature (720p)
+https://cdn-nvm-london.lolokoko.tv/VIASATNATURE.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PowerTV.pl",Power TV (576p)
+https://cdn-nvm-london.lolokoko.tv/powertv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PULS2.pl",Puls 2 (720p)
+https://cdn-nvm-london.lolokoko.tv/TVPuls2.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="STOPKLATKA.pl",Stopklatka TV (720p)
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:125]:
```suggestion
#EXTINF:-1 tvg-id="StopklatkaTV.pl",Stopklatka TV (720p)
```

File: streams/pl_lolokoko.m3u [hunk 10]
```diff
@@ -1,45 +1,176 @@
...
+https://cdn-nvm-london.lolokoko.tv/polsatplay.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PolsatSport.pl",Polsat Sport (720p)
+https://cdn-nvm-london.lolokoko.tv/polsatsport.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PolsatViasatHistory.pl",Polsat Viasat History (720p)
+https://cdn-nvm-london.lolokoko.tv/VIASATHISTORY.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PolsatViasatNature.pl",Polsat Viasat Nature (720p)
+https://cdn-nvm-london.lolokoko.tv/VIASATNATURE.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PowerTV.pl",Power TV (576p)
+https://cdn-nvm-london.lolokoko.tv/powertv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="PULS2.pl",Puls 2 (720p)
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:123]:
```suggestion
#EXTINF:-1 tvg-id="Puls2.pl",Puls 2 (720p)
```

File: streams/pl_lolokoko.m3u [hunk 11]
```diff
@@ -1,45 +1,176 @@
...
+#EXTINF:-1 tvg-id="SuperPolsat.pl",Super Polsat (720p)
+https://cdn-nvm-london.lolokoko.tv/SuperPolsat.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TBNPolska.pl",TBN Polska (720p)
+https://cdn-nvm-london.lolokoko.tv/tbn.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Tele5Niem.pl",Tele5 (720p)
+https://cdn-nvm-london.lolokoko.tv/Tele5.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="teleTOON+.pl",Teletoon (720p)
+https://cdn-nvm-london.lolokoko.tv/TELETOON.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TLC.pl",TLC (720p)
+https://cdn-nvm-london.lolokoko.tv/Tlc.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:138]:
```suggestion
```
DMCA.

File: streams/pl_lolokoko.m3u [hunk 12]
```diff
@@ -1,45 +1,176 @@
...
+#EXTINF:-1 tvg-id="Bollywood.pl",Bollywood (720p)
+https://cdn-nvm-london.lolokoko.tv/bollywood.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Cinemax.pl",Cinemax (720p)
+https://cdn-nvm-london.lolokoko.tv/cinemaxhd.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="DaVinci.pl",DaVinci (576p)
+https://cdn-nvm-london.lolokoko.tv/DaVinci.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="DiscoveryNiem.pl",Discovery (720p)
+https://cdn-nvm-london.lolokoko.tv/discoverychannel.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="DiscoveryLife.pl",Discovery Life (720p)
+https://cdn-nvm-london.lolokoko.tv/DiscoveryLife.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:28]:
```suggestion
```
DMCA.

File: streams/pl_lolokoko.m3u [hunk 13]
```diff
@@ -1,45 +1,176 @@
...
+#EXTINF:-1 tvg-id="DTX.pl",DTX (720p)
+https://cdn-nvm-london.lolokoko.tv/DTX.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="EpicDrama.pl",Epic Drama (720p)
+https://cdn-nvm-london.lolokoko.tv/EpicDrama.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="EskaTV.pl",Eska TV (720p)
+https://cdn-nvm-london.lolokoko.tv/Eskatv.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="EskaTVExtra.pl",Eska TV Extra (720p)
+https://cdn-nvm-london.lolokoko.tv/EskaTVExtra.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Eurosport1.pl",Eurosport 1 (720p)
+https://cdn-nvm-london.lolokoko.tv/EuroSport1.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:42]:
```suggestion
```
DMCA.

File: streams/pl_lolokoko.m3u [hunk 14]
```diff
@@ -1,45 +1,176 @@
...
+#EXTINF:-1 tvg-id="TBNPolska.pl",TBN Polska (720p)
+https://cdn-nvm-london.lolokoko.tv/tbn.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Tele5Niem.pl",Tele5 (720p)
+https://cdn-nvm-london.lolokoko.tv/Tele5.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="teleTOON+.pl",Teletoon (720p)
+https://cdn-nvm-london.lolokoko.tv/TELETOON.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TLC.pl",TLC (720p)
+https://cdn-nvm-london.lolokoko.tv/Tlc.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TTV.pl",TTV (720p)
+https://cdn-nvm-london.lolokoko.tv/Ttv.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:140]:
```suggestion
```
DMCA.

File: streams/pl_lolokoko.m3u [hunk 15]
```diff
@@ -1,45 +1,176 @@
...
+#EXTINF:-1 tvg-id="TVN.pl",TVN (720p)
+https://cdn-nvm-london.lolokoko.tv/TVN.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TVN7.pl",TVN 7 (720p)
+https://cdn-nvm-london.lolokoko.tv/TVN7.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TVN24.pl",TVN 24 (720p)
+https://cdn-nvm-london.lolokoko.tv/Tvn24.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TVN24BiS.pl",TVN 24 Bis (720p)
+https://cdn-nvm-london.lolokoko.tv/Tvn24Bis.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TVNTurbo.pl",TVN Turbo (720p)
+https://cdn-nvm-london.lolokoko.tv/TvnTurbo.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/pl_lolokoko.m3u:158]:
```suggestion
```
DMCA.

**Review by Alstruit — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Comment by R4IN2007** [comment]
Here some Poland streams
Fox became FX & TNT became Warner TV

#EXTINF:-4, AMC | (PL)
https://cdn-notpub-london.lolokoko.tv/amc.stream/chunks.m3u8

#EXTINF:-4, Animal Planet | (PL)
https://cdn-notpub-london.lolokoko.tv/animalplanet.stream/chunks.m3u8

#EXTINF:-4, AXN HD | (PL)
https://cdn-notpub-london.lolokoko.tv/axnpolska.stream/chunks.m3u8

#EXTINF:-4, AXN Black | (PL)
https://cdn-notpub-london.lolokoko.tv/axnblack.stream/chunks.m3u8

#EXTINF:-4, AXN White | (PL)
https://cdn-notpub-london.lolokoko.tv/axnwhite.stream/chunks.m3u8

#EXTINF:-4, AXN Spin HD | (PL)
https://cdn-notpub-london.lolokoko.tv/axnspin.stream/chunks.m3u8

#EXTINF:-4, Cartoon Network | (PL)
https://cdn-notpub-london.lolokoko.tv/cartoonnetwork.stream/chunks.m3u8

#EXTINF:-4, CBS Reality | (PL)
https://cdn-notpub-london.lolokoko.tv/cbsreality.stream/chunks.m3u8

#EXTINF:-4, Cinemax | (PL)
https://cdn-notpub-london.lolokoko.tv/cinemax.stream/chunks.m3u8

#EXTINF:-4, Cinemax 2 | (PL)
https://cdn-notpub-london.lolokoko.tv/cinemax2.stream/chunks.m3u8

#EXTINF:-4, Comedy Central | (PL)
https://cdn-notpub-london.lolokoko.tv/comedycentral.stream/chunks.m3u8

#EXTINF:-4, Discovery Channel | (PL)
https://cdn-notpub-london.lolokoko.tv/discoverychannel.stream/chunks.m3u8

#EXTINF:-4, Discovery Historia | (PL)
https://cdn-notpub-london.lolokoko.tv/discoveryhistoria.stream/chunks.m3u8

#EXTINF:-4, Discovery Life | (PL)
https://cdn-notpub-london.lolokoko.tv/discoverylife.stream/chunks.m3u8

#EXTINF:-4, Discovery Science | (PL)
https://cdn-notpub-london.lolokoko.tv/discoveryscience.stream/chunks.m3u8

#EXTINF:-4, DTX | (PL)
https://cdn-notpub-london.lolokoko.tv/dtx.stream/chunks.m3u8

#EXTINF:-4, Disney Channel | (PL)
https://cdn-notpub-london.lolokoko.tv/disneychannel.stream/chunks.m3u8

#EXTINF:-4, Disney Junior | (PL)
https://cdn-notpub-london.lolokoko.tv/disneyjunior.stream/chunks.m3u8

#EXTINF:-4, Disney XD | (PL)
https://cdn-notpub-london.lolokoko.tv/disneyxd.stream/chunks.m3u8

#EXTINF:-4, Food Network | (PL)
https://cdn-notpub-london.lolokoko.tv/foodnetwork.stream/chunks.m3u8

#EXTINF:-4, FX | (PL)
https://cdn-notpub-london.lolokoko.tv/foxpolska.stream/chunks.m3u8

#EXTINF:-4, FX Comedy | (PL)
https://cdn-notpub-london.lolokoko.tv/foxcomedy.stream/chunks.m3u8

#EXTINF:-4, HBO
https://cdn-notpub-london.lolokoko.tv/hbo.stream/chunks.m3u8

#EXTINF:-4, HBO 2
https://cdn-notpub-london.lolokoko.tv/hbo2.stream/chunks.m3u8

#EXTINF:-4, HBO 3
https://cdn-notpub-london.lolokoko.tv/hbo3.stream/chunks.m3u8

#EXTINF:-4, History | (PL)
https://cdn-notpub-london.lolokoko.tv/history.stream/chunks.m3u8

#EXTINF:-4, History 2 | (PL)
https://cdn-notpub-london.lolokoko.tv/history2.stream/chunks.m3u8

#EXTINF:-4, MTV | (PL)
https://cdn-notpub-london.lolokoko.tv/mtvpolska.stream/chunks.m3u8

#EXTINF:-4, Paramount Network | (PL)
https://cdn-notpub-london.lolokoko.tv/paramountchannel.stream/chunks.m3u8

#EXTINF:-4, Sci Fi | (PL)
https://cdn-notpub-london.lolokoko.tv/scifi.stream/chunks.m3u8

#EXTINF:-4, Sundance TV | (PL)
https://cdn-notpub-london.lolokoko.tv/sundance.stream/chunks.m3u8

#EXTINF:-4, TLC | (PL)
https://cdn-notpub-london.lolokoko.tv/tlc.stream/chunks.m3u8

#EXTINF:-4, Warner TV | (PL)
https://cdn-notpub-london.lolokoko.tv/tnt.stream/chunks.m3u8

#EXTINF:-4, Travel Channel | (PL)
https://cdn-notpub-london.lolokoko.tv/travelchannel.stream/chunks.m3u8

**Comment by trincowski** [comment]
Can you guys load these streams? All of them appear with an Anti-Piracy logo to me. (And the "Oh No No" song.)

**Pull request merged**